simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/station/evse/StateManagerTest.java

```diff
@@ -42,17 +47,26 @@ void setUp() {
     @Test
     void verifyFullStateFlowPlugThenAuthorize() {
         when(stationStoreMock.findEvse(anyInt())).thenReturn(evseMock);
+        when(stationStoreMock.getTxStopPointValues()).thenReturn(new OptionList<>(Collections.emptyList()));
         when(evseMock.findConnector(anyInt())).thenReturn(new Connector(DEFAULT_CONNECTOR_ID, CableStatus.UNPLUGGED, StatusNotificationRequest.ConnectorStatus.AVAILABLE));
 
         stateManager.cablePlugged(DEFAULT_EVSE_ID, DEFAULT_CONNECTOR_ID);
         checkStateIs(WaitingForAuthorizationState.NAME);
         when(evseMock.getEvseState()).thenReturn(new WaitingForAuthorizationState());
 
         stateManager.authorized(DEFAULT_EVSE_ID, DEFAULT_TOKEN_ID);
+
+        verify(stationMessageSenderMock).sendAuthorizeAndSubscribe(anyString(), anyList(), subscriberCaptor.capture());
```

Comment:
can you extract parts with subscriber triggers to methods with meaningful names? it's not clear whether you actually verify something or just want to trigger